<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphics</class>
 <widget class="QMainWindow" name="graphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_calc">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                             background-color: #FF8200;
                             color: black;
                             border: none;
                             border-radius: 10px;

                         }</string>
    </property>
    <property name="text">
     <string>назад</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_y">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   background-color: #FAFAFF;
   color: black;
   border: none;
   border-radius: 10px;
   padding: 0px 20px;
}

QLineEdit:!placeholder {
    color: black; /* Цвет введенного текста */
    font-style: normal; /* Убрать курсив для введенного текста */
}</string>
    </property>
    <property name="placeholderText">
     <string>max y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                             background-color: #FF8200;
                             color: black;
                             border: none;
                             border-radius: 10px;

                         }</string>
    </property>
    <property name="text">
     <string>♻</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_x">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   background-color: #FAFAFF;
   color: black;
   border: none;
   border-radius: 10px;
   padding: 0px 20px;
}

QLineEdit:!placeholder {
    color: black; /* Цвет введенного текста */
    font-style: normal; /* Убрать курсив для введенного текста */
}</string>
    </property>
    <property name="placeholderText">
     <string>min x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_y">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   background-color: #FAFAFF;
   color: black;
   border: none;
   border-radius: 10px;
   padding: 0px 20px;
}

QLineEdit:!placeholder {
    color: black; /* Цвет введенного текста */
    font-style: normal; /* Убрать курсив для введенного текста */
}</string>
    </property>
    <property name="placeholderText">
     <string>min y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_x">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
   background-color: #FAFAFF;
   color: black;
   border: none;
   border-radius: 10px;
   padding: 0px 20px;
}

QLineEdit:!placeholder {
    color: black; /* Цвет введенного текста */
    font-style: normal; /* Убрать курсив для введенного текста */
}</string>
    </property>
    <property name="placeholderText">
     <string>max x</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
